orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/cluster/FindImageFromClusterTask.groovy

```diff
@@ -124,7 +125,8 @@ class FindImageFromClusterTask extends AbstractCloudProviderAwareTask implements
     Map<Location, String> imageIds = [:]
     Set<String> inferredRegions = new HashSet<>()
 
-    if (cloudProvider == 'aws') {
+    Boolean skipRegionDetection = Boolean.TRUE == config.skipRegionDetection
```

Comment:
Odd. Why not just use `true`?

Reply:
I was lazy and lifted this snippet. I've made it better now.